File: Linguagem de Programação I/Extras/Beecrowd/1039.py
import math

while True:
    try:
        R1, X1, Y1, R2, X2, Y2 = map(int, input().split())
        distance = math.sqrt((X2 - X1) ** 2 + (Y2 - Y1) ** 2)
        if R1 >= distance + R2:
            print('RICO')
        else:
            print('MORTO')
    except EOFError:
        break

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1039.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int R1, X1, Y1, R2, X2, Y2;
    while (cin >> R1 >> X1 >> Y1 >> R2 >> X2 >> Y2) {
        double distance = sqrt(pow(X2 - X1, 2) + pow(Y2 - Y1, 2));
        if (R1 >= distance + R2) {
            cout << "RICO" << endl;
        } else {
            cout << "MORTO" << endl;
        }
    }
    return 0;
}